protocol/tbdex_schema_examples.js

```diff
@@ -1,15 +1,15 @@
 let offering = {
   "description": "Buy BTC with USD!",
   "pair": "BTC_USD",
-  "unitPrice": 27000.00,
-  "baseFee": 1.00,
-  "min": 10.00,
-  "max": 1000.00,
+  "unitPrice": "27000.00",
```

Comment:
nice! this looks great! 

only thing I'm curious about is why these types are strings instead of numbers?

Reply:
Precision matters in all financial applications. You don't want quantities to be truncated.

Reply:
oh right, I see there was [discourse in Slack](https://sq-tbd.slack.com/archives/C05B492CW1H/p1686890655830929) about this

makes sense 👍